src/antsibull/data/ansible-setup_py.j2

```diff
@@ -171,6 +173,20 @@ setup(
     license='GPLv3+',
     python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
     packages=['ansible_collections'],
+{% if version.major >= 6 %}
+    exclude_package_data={
+        'ansible_collections': [
+{%   for name, namespace in collections %}
+            '{{ name }}/{{ namespace }}/.gitignore',
+            '{{ name }}/{{ namespace }}/.github/*',
+            '{{ name }}/{{ namespace }}/.azure-pipelines/*',
+            '{{ name }}/{{ namespace }}/changelogs/*',
+            '{{ name }}/{{ namespace }}/docs/*',
```

Comment:
is `docs/` really unused by ansible? This is the only one I think might be good to keep shipping.

Reply:
It's not used by ansible. It still might be useful though; I guess this is something we have to discuss. I do see both arguments for keeping them (having offline docs can be useful) and for removing them (often these contain only redundant docs that ansible-doc can also show, for example community.aws has 180 generated RST files of 6.1 MB which contain the same information that `ansible-doc community.aws.<module-name>` will also show).

Reply:
I tend to agree that if there is anything of value to keep, it's probably the docs -- they are also not as problematic in terms of volume and size as the other bits.

This list is otherwise a good start, although not exhaustive and would miss quite a few hits from https://gist.github.com/dmsimard/4d53b6578f6a18f86013ffa94214af82

The pattern I had used was: ``find ansible_collections | grep -E "\.gitignore|\.github|\.azure-pipelines|\.swp|\.orig|\.DS_Store|\.keep|\.gitkeep|\.vscode|\.pytest_cache|\.yamllint|\.pre-commit-config\.yaml|\.plugin-cache\.yaml|\.pytest_cache|\.mypy_cache|\.zuul\.yaml|\.flake8|\.circleci|\.gitattributes|\.ansible-lint|\.travis\.yml|\.gitlab-ci\.yml|\.idea|\.settings" | sort``

It would feel a bit "dirty" to include these patterns in the setup.py but I figure they must be defined somewhere...